manifests/filebeat.yaml

```diff
@@ -174,6 +174,13 @@ data:
       hosts: ["${LOGSTASH_URL}"]
       ssl: '{{ .ssl | default ( coll.Dict "enabled" "false" ) | data.ToJSON }}'
 {{ end }}
+{{ if ".kubernetes.apiServerExtraArgs.audit-log-path" }}
```

Comment:
Whats the difference between `if ".kubernetes.apiServerExtraArgs.audit-log-path"` and `if .kubernetes.apiServerExtraArgs.audit-log-path` ?

I think this might also need to be scoped only to the infra-filebeat, not the other custom ones

Reply:
The latter throws a malformed yaml error when trying to be parsed. See https://github.com/flanksource/karina/runs/2400317152?check_suite_focus=true#step:4:736

Ok, how do I go about doing that?

Reply:
> The latter throws a malformed yaml error when trying to be parsed.

Ah thats due to the "-"  in audit-log-path

`if and .name == "infra" ".kubernetes.apiServerExtraArgs.audit-log-path" `